OpenRA.Mods.Common/Widgets/Logic/CreditsLogic.cs

```diff
@@ -39,7 +40,7 @@ public CreditsLogic(Widget widget, ModData modData, Action onExit)
 				onExit();
 			};
 
-			engineLines = ParseLines("AUTHORS");
+			engineLines = ParseLines("AUTHORS", false);
```

Comment:
It would be much simpler if you passed the `Stream` instead of a flag.

This could then look like `engineLines = ParseLines(File.OpenRead(Platform.ResolvePath("./AUTHORS")));`.

Reply:
Do this for both engine and mod?